### ISSUE TITLE
astroquery.vizier does not return all the columns specified

### ISSUE THREAD

**Issue by ysBach** [astropy/astroquery#1502]
If I specify too many (N = 16?) columns, the query returns only the first N columns and ignores the other columns:

```python
import astroquery
from astroquery.vizier import Vizier
from astropy.coordinates import SkyCoord
from astropy import units as u

print(astroquery.__version__)
# 0.3.10.dev5533

columns = ['objID', 'RAJ2000', 'DEJ2000',
                            'e_RAJ2000', 'e_DEJ2000',
                            'Ng', 'gmag', 'e_gmag', 'gmagStd',
                            'Nr', 'rmag', 'e_rmag', 'rmagStd',
                            'Ni', 'imag', 'e_imag', 'imagStd',
                            'Nz', 'zmag', 'e_zmag', 'zmagStd',
                            'Ny', 'ymag', 'e_ymag', 'ymagStd'
                            ]

vquery = Vizier(columns=columns, row_limit=-1)

field = SkyCoord(ra=100*u.deg, dec=20*u.deg, frame='icrs')

queried = vquery.query_region(field,
                               radius=0.01*u.deg,
                               catalog="II/349/ps1")[0]

print(len(queried.colnames), len(columns))
# 16 25
```

![image](https://user-images.githubusercontent.com/17214293/60764510-56b23880-a0c6-11e9-9853-e3fe01f34bc8.png)

If I use the identical code but replacing ``columns = ["*"]`` or ``columns=["**"]``, it works as expected.

I encountered with this while doing an urgent work, and thus have not yet tested for other catalogs.

**Comment by keflavich** [comment]
cc @bmatthieu3 is this a problem on our side or is this a limitation built in to the server?

**Comment by philastrophist** [comment]
```
shen11 = Vizier.query_constraints('J/ApJS/194/45', z='> 0.34 & < 0.4', columns=["**"])
assert len(shen11[0].colnames) ==142
```
This fails and only returns the default columns on Vizier, I need all of them!

**Comment by odysseus9672** [comment]
I'm not sure where the problem is, myself, but I can work around it with `columns=["all"]` and then deleting the columns I don't want.

**Comment by hamogu** [comment]
I've independently found this, too. The limit seems to be 14 columns.
```
from astroquery.vizier import Vizier

cols = ['_q', '_d', '_2RXS', 'RAJ2000', 'DEJ2000', 'Cts', 'e_Cts', 'HR1', 'e_HR1', 'HR2', 'e_HR2',
        'CRate', 'e_CRate', 
        'NHfitm', 'e_NHfitm', 'Tempfitm', 'e_Tempfitm', 'Fluxm', 'Chi2rspm']
v = Vizier(columns=cols)
cat_2rxs = Vizier(columns=cols).query_region("Sirius", radius="30s", catalog="J/A+A/588/A103")
print(cat_2rxs)
```

**Comment by jwoillez** [comment]
@philastrophist Sorry for the late feedback. Your syntax is off. All keywords inside the `query_constraint()` call are constraints on columns. The syntax you are after is:

`shen11 = Vizier(catalog='J/ApJS/194/45', columns=['**']).query_constraints(z='> 0.34 & < 0.4')`

**Comment by jwoillez** [comment]
@ysBach With PR #2012, I cannot reproduce the issue and can obtain all 25 requested columns. No idea if this is a change on the `Vizier` side, or a code change on the `astroquery` side.

@keflavich, @bsipocz, I recommend closing this issue, or I mark it as fixed in #2012. Let me know.
